### ISSUE TITLE
hw_CANSocket errors could be shown in the web interface

### ISSUE THREAD

**Issue by DePierre** [eik00d/CANToolz#84]
Hi there,

Nothing urgent there, only a first-time user experience I wanted to share with you.

Trying out with the `hw_CANSocket` module, I noticed that the exceptions raised during errors were only shown in the console but not in the interface. For instance, with the following broken configuration:

```
load_modules = {
    'hw_CANSocket': {'iface':'vcanINVALID', 'speed': 500},
    'mod_stat': {}
}

actions = [
    {'hw_CANSocket': {'action': 'read','pipe': 1}}
    {'mod_stat': {'action': 'read', 'pipe': 1}}
]
```

Clicking on `Start/Stop` will show the following exception in the console:
```
$ python3 cantoolz.py -g w -c can_socket.py


   _____          _   _ _______          _
  / ____|   /\   | \ | |__   __|        | |
 | |       /  \  |  \| |  | | ___   ___ | |____
 | |      / /\ \ | . ` |  | |/ _ \ / _ \| |_  /
 | |____ / ____ \| |\  |  | | (_) | (_) | |/ /
  \_____/_/    \_\_| \_|  |_|\___/ \___/|_/___|



('CANtoolz WEB started at port: ', 4444)
	To exit CTRL-C...
[*] HTTPD: Received connection from 127.0.0.1
[*] HTTPD: Received connection from 127.0.0.1
[*] HTTPD: Received connection from 127.0.0.1
[*] HTTPD: Received connection from 127.0.0.1
[*] HTTPD: Received connection from 127.0.0.1
127.0.0.1 - - [12/Mar/2017 17:46:31] "GET /api/status HTTP/1.1" 200 -
127.0.0.1 - - [12/Mar/2017 17:46:31] "GET /api/status HTTP/1.1" 200 -
127.0.0.1 - - [12/Mar/2017 17:46:31] "GET /api/status HTTP/1.1" 200 -
127.0.0.1 - - [12/Mar/2017 17:46:31] "GET /api/status HTTP/1.1" 200 -
127.0.0.1 - - [12/Mar/2017 17:46:31] "GET /api/status HTTP/1.1" 200 -
127.0.0.1 - - [12/Mar/2017 17:46:31] "GET /api/status HTTP/1.1" 200 -
hw_CANSocket: ERROR: [Errno 19] No such device
127.0.0.1 - - [12/Mar/2017 17:46:31] "GET /api/status HTTP/1.1" 200 -
127.0.0.1 - - [12/Mar/2017 17:46:31] "GET /api/start HTTP/1.1" 200 -
127.0.0.1 - - [12/Mar/2017 17:46:31] "GET /api/start HTTP/1.1" 200 -
127.0.0.1 - - [12/Mar/2017 17:46:31] "GET /api/status HTTP/1.1" 200 -
127.0.0.1 - - [12/Mar/2017 17:46:32] "GET /api/status HTTP/1.1" 200 -
^CPlease wait... (do not press ctr-c again!)
gg bb
pop@debian:~/can/CANToolz$ 
```

But nothing on the web interface. Checking the module status, it shows `Current status: True` but nothing more.

It would be more pertinent to have:

+ The full stack trace in the console (for troubleshooting purpose)
+ The exception name in the web interface (for user-friendliness)

To reproduce, simply run `cantoolz` with the configuration above and click on `Start/Stop` in the web interface.

**Comment by eik00d** [comment]
Hi!

First of all - thank you for the good advises and interest to the project.
I will try to implement both ideas this week.

BR,
Alexey

**Comment by eik00d** [comment]
Fixed at #88
